client/signup/steps/website-content/index.tsx

```diff
@@ -25,6 +27,25 @@ import './style.scss';
 
 const debug = debugFactory( 'calypso:difm' );
 
+const DialogContent = styled.div`
+	padding: 16px;
+	p {
+		font-size: 1rem;
+		color: var( --studio-gray-50 );
+	}
+`;
+
+const DialogButton = styled( Button )`
+	box-shadow: 0px 1px 2px rgba( 0, 0, 0, 0.05 );
+	border-radius: 5px;
+	padding: 10px 64px;
+	--color-accent: #117ac9;
+	--color-accent-60: #0e64a5;
```

Comment:
These should be added automatically by the following sass file. Any reason the variables are defined here again?
https://github.com/Automattic/wp-calypso/blob/364ff7a9b5f5ada58e985ddb27ffe4ca1ad80e52/client/signup/style.scss#L335-L336

Reply:
Yes, these styles are added to the element with the `layout` class, which is a descendant of `<body>`. However, the `Dialog` component adds the modal as a descendant of `<body>` itself and not to the element with the `layout` class.